style.css

```diff
@@ -0,0 +1,102 @@
+/* Generic designing */
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  background-color: #e5e5e5;
+}
+
+/* Header Disigning */
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 20px 0 20px;
+  background-color: #fff;
+}
+
+.logo {
+  height: auto;
+  width: auto;
+  font-family: Poppins, serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #6070ff;
+}
+
+/* Main Disigning */
+
+.main {
+  display: flex;
+  flex-direction: column;
+  background-image: url(/img/header-shapes\ mobile.png);
```

Comment:
- [ ] kindly consider making sure that your background image looks like the one given in the design by fixing the way you are positioning it to make your design an exact duplicate of the given one
- your design 👇 
<img width="366" alt="Screen Shot 2023-07-05 at 09 09 13" src="https://github.com/ali-techworld/Portfolio-Setup/assets/60146030/7dafcde4-0a9c-4359-b9f6-eadf384d2791">

- given design 👇 
<img width="377" alt="Screen Shot 2023-07-05 at 09 10 30" src="https://github.com/ali-techworld/Portfolio-Setup/assets/60146030/00f3ba96-5567-4124-b66c-4788d2d7f716">